pmlc/target/x86/tests/conv_sequence.mlir

```diff
@@ -0,0 +1,188 @@
+// RUN: pmlc-opt  -convert-linalg-to-loops -pxa-reorder-layouts="allow-reorder=true make-user-layouts-explicit=true" -canonicalize  -x86-stencil-tpp-gemm %s | FileCheck %s
+
+
+// PLAIDML_VERBOSE=5 build-x86_64/Release/bin/pmlc-opt -convert-linalg-to-loops -pxa-reorder-layouts="allow-reorder=true make-user-layouts-explicit=true" -canonicalize -x86-stencil-tpp-gemm pmlc/target/x86/tests/conv_sequence.mlir > output
+
+// WithOUT data reordering: build-x86_64/Release/bin/pmlc-opt -convert-linalg-to-loops -canonicalize -x86-stencil-tpp-gemm -x86-convert-pxa-to-affine --normalize-memrefs --simplify-affine-structures  -lower-affine  -canonicalize -convert-scf-to-std -x86-convert-std-to-llvm pmlc/target/x86/tests/conv_sequence.mlir | build-x86_64/Release/bin/pmlc-jit -e conv
+
+
+// WITH data reordering: build-x86_64/Release/bin/pmlc-opt -convert-linalg-to-loops -pxa-reorder-layouts="allow-reorder=true make-user-layouts-explicit=true" -canonicalize -x86-stencil-tpp-gemm -x86-convert-pxa-to-affine --normalize-memrefs --simplify-affine-structures  -lower-affine  -canonicalize -convert-scf-to-std -x86-convert-std-to-llvm pmlc/target/x86/tests/conv_sequence.mlir | build-x86_64/Release/bin/pmlc-jit -e conv
+
+// cat ref | grep -v sizes > ref.cleaned 
+// cat out | grep -v sizes > out.cleaned 
```

Comment:
These comments probably don't belong in this file.

Reply:
I have added additional tests - running the same file and checking for the results. I felt the comments stating how to run the code and a good way to check for correctness, would be may still be useful if someone is investigating it later.